completions/ssh-keygen

```diff
@@ -141,15 +145,25 @@ _comp_cmd_ssh_keygen()
             local pathcmd protocols
             pathcmd=$(type -P -- "$1") && local PATH=${pathcmd%/*}:$PATH
             _comp_compgen -v protocols -x ssh query protocol-version
-            local types='dsa ecdsa ecdsa-sk ed25519 ed25519-sk rsa'
+            local -a types=(
+                dsa ecdsa ecdsa-sk ed25519 ed25519-sk rsa
```

Comment:
Maybe we should remove dsa? OpenSSH [removed support](https://www.openssh.org/releasenotes.html#:~:text=This%20release%20removes%20support%20for%20the%20weak%20DSA%20signature%20algorithm) completely in April 2025

Reply:
Maybe. I suppose we could also drop support for protocol version 1 which is long gone as well, but the difference is that supported protocol versions are queryable, key types are not. Anyway I won't include either in this PR.